.bazelci/presubmit.yml

```diff
@@ -25,7 +25,19 @@ platforms:
     - "-//tests/rbe_repo:rbe_autoconf_config_repos_sh_toolchain_test"
     - "-//tests/rbe_repo:rbe_autoconf_config_repos_no_cc_config_run_in_container_test"
     - "-//tests/rbe_repo:rbe_autoconf_config_repos_no_cc_config_sh_toolchain_test"
-    # docker_autoconfig tests
+    test_flags:
+    - "--action_env=RBE_AUTOCONF_ROOT=/workdir"
+    - "--action_env=PATH"
+    - "--test_output=errors"
+    - "--verbose_failures"
+    - "--extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/latest:toolchain_docker"
```

Comment:
This is wrong. Targets under //configs/ubuntu16_04_clang/latest are not in maintenance and should never be used.
You should always use rbe_autoconfig targets to access configs in bazel_toolchains repo. 
Documentation for (buildkite CI with RBE)[https://docs.bazel.build/versions/master/remote-execution-ci.html#setting-up-the-bazel-ci-for-testing] indicates you should be using an rbe autoconfig rule:
```
# Creates toolchain configuration for remote execution with BuildKite CI
# for rbe_ubuntu1604.
rbe_autoconfig(name = "buildkite_config")
```
The expectation is that BuildKite CI is already configured with the right flags to use RBE with this @buildkite_config target.
If this has changed, then we need an update on the (docs)[https://docs.bazel.build/versions/master/remote-execution-ci.html#setting-up-the-bazel-ci-for-testing] and should still be using @buildkite_config targets here.

Reply:
It was not my intention to change this behavior in this PR. I just split the existing task into two.
However, given that these flags are deprecated, it's better that I don't.
I'm going to abandon this PR in favor of https://github.com/bazelbuild/bazel-toolchains/pull/671.